### PR TITLE
Fixed Polygon symbol for alert.

### PR DESCRIPTION
**Pull request by teamagam3**
Made some code changes to remove tranformFromSuperclass method.

### CODE REVIEW

File: app/src/main/java/com/teamagam/gimelgimel/app/map/esri/graphic/EsriSymbolCreationVisitor.java [hunk 1]
```diff
@@ -114,11 +116,21 @@ public void visit(SensorSymbol symbol) {
     }
 
     @Override
-    public void visit(AlertSymbol symbol) {
+    public void visit(AlertPointSymbol symbol) {
         mEsriSymbol = createPictureMarker(R.drawable.ic_alert, DEFAULT_TINT_COLOR);
     }
 
     @Override
+    public void visit(AlertPolygonSymbol symbol) {
```
Comment (gRaytan) [inline app/src/main/java/com/teamagam/gimelgimel/app/map/esri/graphic/EsriSymbolCreationVisitor.java:125]: Tthe idea is that we only have point Ymbol and Polygon Symbol right?

**Review by gRaytan — COMMENTED** [review]
Looks great

File: app/src/main/java/com/teamagam/gimelgimel/app/map/esri/graphic/EsriSymbolCreationVisitor.java [hunk 2]
```diff
@@ -114,11 +116,21 @@ public void visit(SensorSymbol symbol) {
...
+        SimpleFillSymbol simpleFillSymbol = new SimpleFillSymbol(ALERT_TINT_COLOR);
```
Comment (teamagam5) [inline app/src/main/java/com/teamagam/gimelgimel/app/map/esri/graphic/EsriSymbolCreationVisitor.java]: ren simpleFillSymbol to symbol

Comment (teamagam5) [reply app/src/main/java/com/teamagam/gimelgimel/app/map/esri/graphic/EsriSymbolCreationVisitor.java]: and extract method "createPolygonSymbol(int fillColor)" from the duplicate code

File: data/src/main/java/com/teamagam/gimelgimel/data/map/adapter/GeoEntityDataMapper.java [hunk 3]
```diff
@@ -123,10 +151,19 @@ public void visit(SensorEntity entity) {
...
             mGeoContentData = new GeoContentData(
-                    mGeometryMapper.transformToDataSuperclass(entity.getGeometry()),
-                    entity.getText());
+                    mGeometryMapper.transformToData(entity.getGeometry()),
+                    entity.getText()
+            );
+        }
+
+        @Override
+        public void visit(AlertPolygonEntity entity) {
```
Comment (teamagam5) [inline data/src/main/java/com/teamagam/gimelgimel/data/map/adapter/GeoEntityDataMapper.java:162]: same impl. as the method above. extract...?

**Review by teamagam5 — APPROVED** [review]

File: app/src/main/java/com/teamagam/gimelgimel/app/map/esri/graphic/EsriSymbolCreationVisitor.java (hunk 1 above)
Comment (teamagam3) [reply app/src/main/java/com/teamagam/gimelgimel/app/map/esri/graphic/EsriSymbolCreationVisitor.java:125]: Yes

**Pull request merged**